### PR TITLE
Add backup_retention_period variable for teamleader rds read replica

### PR DESCRIPTION
**Pull request by YorickH**

### CODE REVIEW

**Review by duboisph — DISMISSED** [review]

**Comment by duboisph** [comment]
LGTM, could you just update the documentation too please?

File: rds-replica/variables.tf [hunk 1]
```diff
@@ -73,3 +73,7 @@ variable "enabled_cloudwatch_logs_exports" {
   default     = []
 }
 
+variable "backup_retention_period" {
+  description = "How long do you want to keep RDS Slave backups"
```
Comment (duboisph) [inline rds-replica/variables.tf:77]: You should also specify the type here (`number`)

File: rds-replica/variables.tf [hunk 2]
```diff
@@ -73,3 +73,7 @@ variable "enabled_cloudwatch_logs_exports" {
...
+  default     = "14"
```
Comment (duboisph) [inline rds-replica/variables.tf]: No quotes needed for a `number`

**Review by duboisph — CHANGES_REQUESTED** [review]
Just some small change needed + could you also update the README please?

**Comment by YorickH** [comment]
@duboisph updated

**Review by duboisph — APPROVED** [review]

**Review by iuriaranda — CHANGES_REQUESTED** [review]
Can you also update the documentation in the README with the new variable?

**Review by iuriaranda — APPROVED** [review]
Great. Thanks!

**Comment by iuriaranda** [comment]
Released in version 5.0.1

Thanks for the contribution 😉

**Pull request merged**